.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           command: npm run lint
       - run:
           name: "Build files"
-          command: npm run build
+          command: npm run prepare
```

Comment:
We shouldn't using the `prepare` script directly. NPM runs that script on the following conditions:
>**prepare**: Run both BEFORE the package is packed and published, on local `npm install` without any arguments, and when installing git dependencies (See below). This is run AFTER `prepublish`, but BEFORE `prepublishOnly`.

Added some notes to the `package.json` how we should build it.

Reply:
ahh interesting. fixed